test/screenshot/spec/mdc-dialog/classes/baseline-simple.html

```diff
@@ -34,12 +34,12 @@
     <link rel="stylesheet" href="../../../out/spec/mdc-dialog/fixture.css">
   </head>
 
-  <body class="test-container test-container--edge-fonts">
+  <body class="test-container test-container--edge-fonts mdc-dialog-scroll-lock">
     <main class="test-viewport test-viewport--center">
 
       <button class="test-dialog-open-button" data-test-dialog-id="test-dialog" autofocus>Open Dialog</button>
 
-      <div class="mdc-dialog test-dialog"
+      <div class="mdc-dialog mdc-dialog--open test-dialog test-dialog--simple"
```

Comment:
What is the --simple class doing? This is the only place I see it. Ditto the other --simple classes later in this page.

I noticed you removed --alert classes in another file, should these also be removed?

Reply:
I split this PR out from #3602. Looks like I missed a class :-)

Reply:
Removed